doc/source/.res/replace.txt

```diff
@@ -20,5 +20,6 @@
 .. |rhel| replace:: RHEL
 
 .. |version.next| replace:: 9.0
+.. |changed-version| replace:: 8.0.15-5
```

Comment:
???

Reply:
I'll remove it. It was used in an earlier version of the tablespace encryption doc.